### PR TITLE
refactoring sighting list to pull let data on load

### PR DESCRIPTION
**Pull request by wallace-y**

### CODE REVIEW

**Comment by wallace-y** [comment]
@Rim0Rad I started this branch before the rework branch so there might be some working out we need to do after merging the rework. The main part of the merge is fixing the data being called though

**Review by Rim0Rad — APPROVED** [review]
This commit looks good, Dont merge any changes to login and HomePage though

**Pull request merged**
